curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/check-for-all-or-none.english.md

```diff
@@ -32,13 +32,13 @@ Change the regex <code>favRegex</code> to match both the American English (favor
 ```yml
 tests:
   - text: Your regex should use the optional symbol, <code>?</code>.
-    testString: assert(favRegex.source.match(/\?/).length > 0);
+    testString: favRegex.lastIndex = 0; assert(favRegex.source.match(/\?/).length > 0);
```

Comment:
```suggestion
    testString: assert(favRegex.source.match(/\?/).length > 0);
```

Reply:
True, but it doesn't do any harm, does it?

Reply:
No, it really doesn't - and since the user here isn't fixing it. I say we just go with it.